packages/core/src/tests/html-table/striped/index.html

```diff
@@ -15,7 +15,7 @@
     <title>Stencil Component Starter</title>
   </head>
   <body>
-    <table class="table table-striped">
+    <table class="ix-table table-striped">
```

Comment:
Should this not be prefixed as well?
```suggestion
    <table class="ix-table ix-table-striped">
```